packages/extension-ui/src/Popup/ImportQr.tsx

```diff
@@ -50,8 +52,12 @@ export default function ImportQr(): React.ReactElement {
           .then(() => onAction('/'))
           .catch((error: Error) => console.error(error));
       }
+
+      if (name && password) {
```

Comment:
in other places we change `onAction('/')` to step change, will it work correctly here?

Reply:
This entire qr code importing thing is not used in our repo (dead code kek). And since it's not used, we're not opening it in a new window and hence no need of changing `onAction('/'). There are also no steps in this module.

I only refactored this file to adapt to the change in `AccountNamePasswordCreation.tsx` in which I extracted some of the logic making it more generic.